temporal-testing-junit4/src/main/java/io/temporal/testing/TestWorkflowRule.java

```diff
@@ -155,7 +158,13 @@ public Builder setWorkerOptions(WorkerOptions options) {
     }
 
     public Builder setWorkerFactoryOptions(WorkerFactoryOptions options) {
-      this.workerFactoryOptions = options;
+      this.workerFactoryOptions =
+          (options.getWorkerInterceptors() == null)
```

Comment:
I don't think getWorkerInterceptors ever returns null: https://github.com/temporalio/sdk-java/blob/b3797d6f0dcf174731dd1c5596a3a2e4bcaf2dbb/temporal-sdk/src/main/java/io/temporal/worker/WorkerFactoryOptions.java#L182

It looks like there is no unit test that validates both branches of this condition.

Reply:
Note "validate" param in the constructor. By default it's false and workerInterceptors is null